kea-vscode/src/views/common/file-tree-node.ts

```diff
@@ -40,6 +43,20 @@ export class FileTreeNode implements IParentTreeNode<ReviewCommentTreeNode> {
     treeItem.iconPath = this.#iconPath;
     treeItem.tooltip = this.#tooltip;
 
+    if (this.#ctx) {
+      treeItem.command = this.#ctx.commandManager.getCommand(
+        "kea.openCommitFileDiff",
+        "Open File Diff",
+        { resourceUri: this.#resourceUri }
+      );
+    } else {
+      treeItem.command = {
+        command: "kea.openCommitFileDiff",
+        title: "Open File Diff",
+        arguments: [{ resourceUri: this.#resourceUri }],
+      };
```

Comment:
I get the following:

```log
2025-07-20 15:34:33.702 [error] Failed to parse decoration payload Failed to parse decoration payload
2025-07-20 15:34:34.392 [error] Failed to parse decoration payload Unknown decoration scheme: kea-commit-file
```

I also get the following error:

`Actual command not found, wanted to execute kea.openCommitFileDiff /47`